### PR TITLE
Added specialised IrisGribTest.get_result_path

### PR DESCRIPTION
**Pull request by bjlittle**
@pelson as per https://github.com/SciTools/iris/pull/2037

### CODE REVIEW

File: iris_grib/tests/__init__.py [hunk 1]
```diff
@@ -84,3 +87,6 @@ def result_path(self, basename=None, ext=''):
                               self.__class__.__name__.replace('Test_', ''),
                               filename)
         return result
+
+
+get_result_path = IrisGribTest.get_result_path
```
Comment (pelson) [inline iris_grib/tests/__init__.py]: Let's not add this.

**Comment by pelson** [comment]
👍

File: iris_grib/tests/__init__.py (hunk 1 above)
Comment (bjlittle) [inline iris_grib/tests/__init__.py]: Agreed.

**Comment by bjlittle** [comment]
@pelson Travis may barf for the `python=27 & iris=1.9` matrix case, but `python=27 & iris=master` should be okay ...

**Pull request merged**